crates/matrix-sdk-ui/src/timeline/inner/state.rs

```diff
@@ -790,20 +790,25 @@ pub(in crate::timeline) struct TimelineInnerMetadata {
     /// List of all the events as received in the timeline, even the ones that
     /// are discarded in the timeline items.
     pub all_events: VecDeque<EventMeta>,
+
     next_internal_id: u64,
     pub reactions: Reactions,
     pub poll_pending_events: PollPendingEvents,
     pub fully_read_event: Option<OwnedEventId>,
-    /// Whether the fully-read marker item should try to be updated when an
+
+    /// Whether we should try to update the fully-read marker item when a new
     /// event is added.
-    /// This is currently `true` in two cases:
+    ///
+    /// This happens when:
     /// - The fully-read marker points to an event that is not in the timeline,
     /// - The fully-read marker item would be the last item in the timeline.
     pub event_should_update_fully_read_marker: bool,
+
     pub read_receipts: ReadReceipts,
-    /// the local reaction request state that is queued next
+
+    /// The local reaction request state that is queued next.
     pub reaction_state: IndexMap<AnnotationKey, ReactionState>,
-    /// the in flight reaction request state that is ongoing
+    /// The inflight reaction request state that is ongoing.
```

Comment:
Not `in-flight`?

Reply:
in-flight seems more normal to me. https://dictionary.cambridge.org/dictionary/english/in-flight